proposals/nnnn-main-menu-tiles.md

```diff
@@ -0,0 +1,234 @@
+# Add Tiles as an Option for Main Menus
+
+* Proposal: [SDL-NNNN](NNNN-main-menu-tiles.md)
+* Author: [Joel fischer](https://github.com/joeljfischer)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Java Suite / Cloud Proxy / RPC / HMI]
```

Comment:
The Cloud Proxy is now included in the Java Suite library, so I believe this can be removed as an Impacted Platform.

Reply:
Fixed!